docs/version0.19.md

```diff
@@ -30,6 +30,7 @@
 - [New binaries and changes to supported environments](#binaries-and-supported-environments)
 - [Option to print code cache usage on stderr at JVM shutdown](#option-to-print-code-cache-usage-on-stderr-at-JVM-shutdown)
 - [![Start of content that applies to Java 8](cr/java8.png) `StringBuffer` and `StringBuilder` above 1 G grow to the maximum size](stringbuffer-and-stringbuilder-above-1-g-grow-to-the-maximum-size)
+- [The default value of `-Xmx` has been changed in Java 8](#the-default-value-of-max-heapsize-has-been-changed-in-java8)
```

Comment:
Please add the icon to this bullet as per the previous one. i.e.  [![Start of content that applies to Java 8](cr/java8.png)

Please use current tense. No need to state Java 8 as you used an icon. Also please update the link, which currently does not work.

The default value of `-Xmx` is changed

Reply:
done, with the icon the link seems not work.

Reply:
It's not the icon - you need to remove the extra - in the link next to -xmx (#the-default-value-of--xmx-is-changed). It's a quirk of the build. Alternative is to state default maximum heap size, which would give the user a clearer understanding of what the change is about (unless they are familiary with -Xmx)